### PR TITLE
adding in docs to create/update ticketing orders via SDK

### PR DESCRIPTION
**Pull request by patmanh**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -107,19 +113,23 @@ HIVE_SDK(
 }
 ```
 
-This call will take the provided user data, and use it to authenticate the current user and add them to your brand's contact list in Hive.
+This call will take the provided user data, and use it to "authenticate" the current user and add them to your brand's contact list in Hive.
 
 The <code>email</code> field is the only required field, although passing along as much data as possible is recommended. For instance, including <code>firstName</code> will help to generate a gender for your contacts in Hive. <code>location</code> should be a string with as much location granularity as possible (we'll take care of geocoding, etc. on our own).
 
+If you're including a user's phone number, it is recommended to provide phone numbers in E.164 or RFC3966 format to ensure that phone numbers are properly saved, . In cases where Hive cannot resolve a poorly formatted phone number, a default country code of `+1` (United States/Canada) will be assumed.
```
Comment (prateekmadhikar) [inline source/index.html.md]:
```suggestion
If you're including a user's phone number, it is recommended to provide phone numbers in E.164 or RFC3966 format to ensure that phone numbers are properly saved. In cases where Hive cannot resolve a poorly formatted phone number, a default country code of `+1` (United States/Canada) will be assumed.
```

Comment (prateekmadhikar) [reply source/index.html.md]: add a note/warning about the implications of assuming a +1 that may result in a user not getting saved at all?

Comment (prateekmadhikar) [reply source/index.html.md]: also side note: maybe we should enforce E.164 formatting and not make any assumptions at least with the sdk?

File: source/index.html.md [hunk 2]
```diff
@@ -131,7 +141,10 @@ HIVE_SDK(
     firstName: 'Patrick',  // optional
     lastName: 'Hannigan',  // optional
 
+    firstName: 'Patrick',  // optional
+    lastName: 'Hannigan',  // optional
```
Comment (prateekmadhikar) [inline source/index.html.md]: this is already there in the lines above!

File: source/index.html.md [hunk 3]
```diff
@@ -193,9 +204,105 @@ HIVE_SDK(
...
+```
+
+All fields are required. If a field isn't provided, we'll throw an error message.\
+
+All dates (ie <code>event.start_time</code> etc.) should be provided as strings in ISO 8601 format. If you dont specify a timezone offset in your date, please make sure the time you're passing is in UTC.
+
+Make sure the values for <code>id</code> and <code>event.id</code> are unique per order/event and match what you'd save into your database. You'll use the same order <code>id</code> when updating this order in the future (see the update command below). We use <code>event.id</code>'s to keep track of orders made by different people to the same event.
+
+Ensure the order's <code>status</code> is correctly provided. Currently, there are two values for <code>status</code> that are supported:
+  - <code>started</code> Use this when a user has created an order but not yet completed it. We'll store the "in progress" order inside Hive.co and you can use this data to e.g. send emails to users who haven't completed their orders.
```
Comment (prateekmadhikar) [inline source/index.html.md]:
```suggestion
  - <code>started</code> Use this when a user has created an order but not yet completed it. We'll store the "in progress" order inside Hive.co and you can use this data to target these users, e.g. for sending them abandoned cart reminder emails.
```

File: source/index.html.md [hunk 4]
```diff
@@ -193,9 +204,105 @@ HIVE_SDK(
...
+  - <code>completed</code> Use this when a user has fully completed an order (payment, etc).
```
Comment (prateekmadhikar) [inline source/index.html.md]:
```suggestion
  - <code>completed</code> Use this when a user has fully completed an order, i.e. when the order has been confirmed to be paid for.
```

File: source/index.html.md [hunk 5]
```diff
@@ -193,9 +204,105 @@ HIVE_SDK(
...
+  function(data){
+    // failure callback, called if something goes wrong
+    // error information is provided in the "data" param
+  }
+);
+```
+
+Orders saved in Hive.co should be kept up-to-date throughout a user's purchasing session (e.g. as they add & remove items from their cart and eventually checkout and complete the order).
+
+All fields are required. If a field isn't provided, we'll throw an error message.
```
Comment (prateekmadhikar) [inline source/index.html.md]: `total_paid` isn't required in update calls

File: source/index.html.md [hunk 6]
```diff
@@ -193,9 +204,105 @@ HIVE_SDK(
...
+
+Make sure the value for the order's <code>id</code> matches what you used when making the previous create command. If you provide an order <code>id</code> that doesn't exist yet, we'll discard the command.
```
Comment (prateekmadhikar) [inline source/index.html.md]:
```suggestion
Make sure the value for the order's <code>id</code> matches what you used when making the previous create command. If you provide an order <code>id</code> that doesn't exist yet, we'll discard the order update.
```

**Review by prateekmadhikar — COMMENTED** [review]
few suggestions/comments but lgtm otherwise!

File: source/index.html.md (hunk 1 above)
Comment (patmanh) [reply source/index.html.md]: will update copy - part of me likes handling "fuzzy" phone numbers a bit b/c it takes a bit of the onus off the dev implementing our SDK to enforce valid PN's etc.

maybe we could add our own "phone number poorly formatted" error callback the SDK could provide that implementers could react to 💭

**Pull request merged**